src/appSelector.ts

```diff
@@ -332,25 +359,28 @@ function manifestDevicesToLedgerDevices(manifestDevices: string): LedgerDevice[]
         case "stax":
           return "Stax";
         case "flex":
-            return "Flex";          
+            return "Flex";
         default:
           throw new Error("Invalid device in manifest : " + device);
       }
     });
 }
 
-// Get the app name from the Makefile (for C apps)
-function getAppNameFromMakefile(content: string): string {
-  let appName: string;
-  // Find the app name in the Makefile
-  const regex = new RegExp(`${C_APP_NAME_MAKEFILE_VAR}\\s*=\\s*(.*)`);
-  const match = content.match(regex);
-  if (match) {
-    appName = match[1];
-  } else {
-    throw new Error("No app name found in Makefile");
+// Get the app name (for C apps)
+function getAppName(appdir: string): string {
+  let optionsExecSync: cp.ExecSyncOptions = { stdio: "pipe", encoding: "utf-8" };
+  // If platform is windows, set shell to powershell for cp exec.
+  if (platform === "win32") {
```

Comment:
Has this new feature been tested on windows ?

Reply:
Not yet, but it is a copy-paste of another function `getAndBuildAppTestsDependencies`.